docs/setup/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-title: Setup lakeFS
+title: Set up lakeFS
```

Comment:
This is a parent of other pages: it needs to be changed in the parent directive of all its children

Reply:
I'm not a native English speaker so I don't know which one is better. My point is that this way the subpages will not appear - so the PR either contains the whole thing or none.